main/modes/platformer/mode_platformer.c

```diff
@@ -260,13 +613,7 @@ void platformerExitMode(void)
  */
 void platformerMainLoop(int64_t elapsedUs)
 {
-    // Execute logic at 20fps
-    platformer->frameTimer += elapsedUs;
-    if (platformer->frameTimer >= 50000)
-    {
-        platformer->frameTimer -= 50000;
-        platformer->update(platformer);
-    }
+    platformer->update(platformer);
```

Comment:
As a caveat, the system is gonna try to call this as close to 60fps as possible, but it's not _guaranteed_. It's still safer to respect `elapsedUs`, and I'd go so far as to recommend putting this back and checking it against `16666` too. I'd even throw a `while` in here too
```C
    platformer->frameTimer += elapsedUs;
    while (platformer->frameTimer >= 16666)
    {
        platformer->frameTimer -= 16666;
        platformer->update(platformer);
    }
```

Reply:
@JVeg199X I'm gonna merge this as-is, and you can make this change if you so choose later.